influxql/INFLUXQL.md

```diff
@@ -687,14 +702,22 @@ unary_expr       = "(" expr ")" | var_ref | time_lit | string_lit | int_lit |
 ## Other
 
 ```
-dimension         = expr .
+alias            = "AS" identifier .
+
+db_name          = identifier .
 
-dimensions        = dimension { "," dimension } .
+dimension        = expr .
+
+dimensions       = dimension { "," dimension } .
+
+field_key        = identifier .
 
 field            = expr [ alias ] .
 
 fields           = field { "," field } .
 
+fill_option      = "null" | "none" | "previous" | int_lit | float_lit .
```

Comment:
I wasn't sure if the string literal fill options should have been proper tokens; they weren't tokens [in the code](https://github.com/influxdb/influxdb/blob/418c8cd6f4dbafccf4484ca67af562fdbbae10e2/influxql/parser.go#L1962-L1971).